libraries/AP_Logger/LogStructure.h

```diff
@@ -76,7 +76,9 @@ const struct UnitStructure log_Units[] = {
     { 'W', "Watt" },        // Watt
     { 'Y', "us" },            // pulse width modulation in microseconds
     { 'z', "Hz" },            // Hertz
-    { '#', "instance" }       // (e.g.)Sensor instance number
+    { '#', "instance" },      // (e.g.)Sensor instance number
+    { '0', "ma.h" },          // milliamp hour
```

Comment:
No way "0" is a good designator for this unit.

Or 1 for watt-hour.

Reply:
open to suggestions, were rather running out of characters.

Reply:
X for Watt hours